packages/service-worker/src/push.ts

```diff
@@ -81,6 +81,7 @@ import {ERR_SW_NOT_SUPPORTED, NgswCommChannel, PushEvent} from './low_level';
  * <code-example path="service-worker/push/module.ts" region="subscribe-to-notification-clicks"
  * header="app.component.ts"></code-example>
  *
+ * @see {@link guide/service-worker-notifications Service worker notifications guide}
```

Comment:
Maybe be more explicit here with something like:

```suggestion
You can read more on handling notification clicks in the [Service worker notifications guide](guide/service-worker-notifications).

```